lms/static/sass/base/_variables.scss

```diff
@@ -134,6 +134,72 @@ $green-u1: desaturate($green,15%);
 $green-u2: desaturate($green,30%);
 $green-u3: desaturate($green,45%);
 
+// copied from cms/static/sass/_variables.scss
+$yellow: rgb(255, 252, 221);      // yellow color used by LMS
+//$yellow: rgb(237, 189, 60);     // yellow color used by Studio
```

Comment:
Curiously, neither is the right yellow according to our recently updated pattern library work. :) (http://ux.edx.org/elements/colors/) I will talk to the UX team and get their thoughts today. thanks for flagging this.